### DISCUSSION TITLE
0.4.2 (2021-11-19)

### DISCUSSION THREAD

**Discussion by diafour** [flant/grafana-statusmap#235] [General]
## What's Changed

* Compatibility with Grafana 8.1+. (#230, #219, #220, #221)
* Fix 'as zero' for spectrum mode. (#209)

**Full Changelog**: https://github.com/flant/grafana-statusmap/compare/v0.4.1...v0.4.2

<hr /><em>This discussion was created from the release <a href='https://github.com/flant/grafana-statusmap/releases/tag/v0.4.2'>0.4.2 (2021-11-19)</a>.</em>

**Comment by diafour** [comment]
The publishing process is changed, no need to make a pull request into 'grafana-plugin-repository'. I've made a request via grafana.com service, but its status is not publicly available. I'll update this discussion when the update is accepted by the Grafana team.

**Reply by diafour** [reply]
Some problems with signing... new archive is uploaded.

**Reply by malmor** [reply]
Hey, are there any updates on this? ✌
We are planning to update our Grafana instance to v8 and currently waiting for this version 0.4.2 to be released on [grafana.com](https://grafana.com/grafana/plugins/flant-statusmap-panel/). 🚀

**Reply by shurup** [reply]
> Hey, are there any updates on this? v We are planning to update our Grafana instance to v8 and currently waiting for this version 0.4.2 to be released on [grafana.com](https://grafana.com/grafana/plugins/flant-statusmap-panel/). rocket

Hi there! We are also waiting for this moment. The v0.4.2 release was submitted for review on November 19th via our organization at grafana.com. Nothing happened since that and, sadly, we can't do anything about this…

**Reply by malmor** [reply]
Okay, thanks for the info!
Then I guess we just have to wait a (little) bit longer 😉

**Reply by diafour** [reply]
I have an update from the Grafana team. They're short in resources to review plugins. Hence, there is no estimate. I've added a docker compose configuration, it might help.

**Reply by shurup** [reply]
@malmor hi! It has finally arrived: https://grafana.com/grafana/plugins/flant-statusmap-panel/

**Reply by malmor** [reply]
@shurup thanks for the info! 👍
